### PR TITLE
Bo

### PR DESCRIPTION
**Pull request by toddr**

### CODE REVIEW

File: ext/B/t/strict.t [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#!./perl -w
+
+use strict;
```
Comment (toddr) [inline ext/B/t/strict.t:3]: Strict GOOD!

File: ext/B/B.pm [hunk 2]
```diff
@@ -6,11 +6,16 @@
...
 #
 package B;
-use strict;
 
-require Exporter;
 @B::ISA = qw(Exporter);
 
+# If B is loaded without imports, we do not want to unnecessarily pollute the stash with Exporter.
+sub import {
+    return unless scalar @_ > 1; # Called as a method call.
```
Comment (toddr) [inline ext/B/B.pm:14]: Blah!

File: ext/B/B.pm [hunk 3]
```diff
@@ -6,11 +6,16 @@
...
+    require Exporter;
+    B->export_to_level(1, @_);
+}
```
Comment (toddr) [inline ext/B/B.pm:17]: Blah Blah!

File: ext/B/B.pm [hunk 4]
```diff
@@ -74,12 +79,12 @@ push @B::EXPORT_OK, (qw(minus_c ppname save_BEGINs
 
 @B::SPECIAL::ISA = 'B::OBJECT';
 
-@B::optype = qw(OP UNOP BINOP LOGOP LISTOP PMOP SVOP PADOP PVOP LOOP COP
+our @optype = qw(OP UNOP BINOP LOGOP LISTOP PMOP SVOP PADOP PVOP LOOP COP
```
Comment (toddr) [inline ext/B/B.pm:82]: use vars is cool!

**Review by toddr — COMMENTED** [review]
Yada Yada  blah

File: ext/B/B.pm (hunk 4 above)
Comment (atoomic) [reply ext/B/B.pm:82]: indeed

**Pull request merged**